recipe/meta.yaml

```diff
@@ -61,6 +63,17 @@ outputs:
     # If only_build_numpy_base: "yes", build numpy-base only; otherwise build all the outputs.
     {% if only_build_numpy_base != 'yes' %}
     test:
+      # requires was added in order to test the dependencies
+      requires:
+        - pytest  # added for testing
+        - pytest-timeout  # added for testing
+        - pytest-xdist  # added for testing
+        - cffi  # added for testing
+        - cython  # added for testing
+        - hypothesis  # added for testing
+        - pytz  # added for testing
+        - setuptools <60.0.0  # added for testing
+        - typing_extensions  # added for testing
```

Comment:
I'm not sure if `test/requires `is needed here if we don't run the pytest command

Reply:
the unnecessary dependencies were removed.